1983 new bulk files

mlissner opened the pull request:
This fixes: #1983 and #1937. 

The general idea, as explained on our call is to use the COPY command to generate CSVs. I also:

 - overhauled the documentation (wasn't really a choice)
 - added an export of our schema to S3
 - yanked out all our old bulk data code and tests

Once this is merged, I'll create a CronJob in k8s to run the script.

mlissner commented:
I just added a tweak to our documentation to add alerts info for the API. Hopefully it's not too much in a single PR!

flooie commented:
Personally, I think the bulk data page is too narrow.  Something about the big empty columns on the sides --- and then big margin in the main column seems weird to me today.

flooie commented on cl/api/templates/rest-docs-vlatest.html:
```diff
@@ -971,6 +971,15 @@ <h4>Security of RECAP Fetch API</h4>
     </p>
 
+    <h3 id="alerts-endpoint">{% url "alert-list" version="v3" %}</h3>
+    <p><a href="{% url "alert_help" %}#search-alerts">Search alerts</a> can be made, viewed, modified, and deleted using this endpoint. This can be extremely useful for organizations that need alerts on many topics or on a changing array of topics. To create an alert, send:
+    </p>
+    <ul>
+      <li><strong><code>name</code></strong> &mdash; A human-friendly name for the alert.</li>
+      <li><strong><code>query</code></strong> &mdash; The URL parameters you would get when creating the alert on the front end.</li>
+      <li><strong><code>rate</code></strong> &mdash; How frequently you want your alert query to be checked and alert emails to be sent.</li>
```
This needs an example post imo.  Also - a note that some frequencies are only available to some people might be smart?

flooie commented on scripts/make_bulk_data.sh:
```diff
@@ -0,0 +1,246 @@
+	       case_name, case_name_full, slug, docket_number, docket_number_core,
+	       pacer_case_id, cause, nature_of_suit, jury_demand, jurisdiction_type,
+	       appellate_fee_status, appellate_case_type_information, mdl_status,
+	       filepath_local, filepath_ia, filepath_ia_json, date_blocked, blocked,
```
`filepath_local` should this be included.

mlissner replied:
Yeah, not a super useful field. Removed.

flooie commented on scripts/make_bulk_data.sh:
```diff
@@ -0,0 +1,246 @@
+	       scdb_votes_minority, source, procedural_history, attorneys,
+	       nature_of_suit, posture, syllabus, headnotes, summary, disposition,
+	       history, other_dates, cross_reference, correction, citation_count,
+	       precedential_status, date_blocked, blcoked, docket_id
```
Typo in `blocked` 

Should we include `date_blocked`? and blocked.

mlissner replied:
Including date_blocked is fine. Nothing too interesting about it, really.

flooie commented on scripts/make_bulk_data.sh:
```diff
@@ -0,0 +1,246 @@
+	       name_middle, name_last, name_suffix, date_dob, date_granularity_dob,
+	       date_dod, date_granularity_dod, dob_city, dob_state, dob_country,
+	       dod_city, dod_state, dod_country, gender, religion, ftm_total_received,
+	       ftn_eid, has_photo, is_alias_of_id
```
Typo in `ftn_eid` should be `ftm_eid`

flooie commented on scripts/make_bulk_data.sh:
```diff
@@ -0,0 +1,246 @@
+	  'set statement_timeout to 0;
+	   COPY people_db_retentionevent (
+	       id, date_created, date_modified, retention_type, date_retention,
+	       votes_yes, votes_no, vote_yes_percent, votes_no_percent, unopposed, won
```
`vote_yes_percent` should be `votes_yes_percent`

flooie reviewed (changes requested):
A handful of typos, and some style suggestions for the bulk page. 

Additionally I think you should expand the example for api with respect to creating alerts.  Perhaps an explanation about turning on and off and frequency.

mlissner commented:
I fixed the typos, thanks. 

For the width, I think the narrow is more readable ([see here](https://baymard.com/blog/line-length-readability)), but I agree it looks weird. A tweak would help it look better, but for now I'm leaning into readability over appearance.  

I added an example for the API endpoint.

Want to merge if this looks good to you?

flooie reviewed (approved):
Looks great.
